massa-ledger/src/ledger_db.rs

```diff
@@ -0,0 +1,273 @@
+// Copyright (c) 2022 MASSA LABS <info@massa.net>
+
+use massa_hash::{Hash, HASH_SIZE_BYTES};
+use massa_models::{Address, Amount, DeserializeCompact, SerializeVarInt};
+use rocksdb::{
+    ColumnFamilyDescriptor, Direction, IteratorMode, Options, ReadOptions, WriteBatch, DB,
+};
+use std::collections::BTreeMap;
+
+use crate::{ledger_changes::LedgerEntryUpdate, LedgerEntry, SetOrDelete, SetOrKeep};
+
+// TODO: remove rocks_db dir when sled is cut out
+const DB_PATH: &str = "../massa-node/storage/ledger/rocks_db";
+const LEDGER_CF: &str = "ledger";
+const METADATA_CF: &str = "metadata";
+const OPEN_ERROR: &str = "critical: rocksdb open operation failed";
+const CRUD_ERROR: &str = "critical: rocksdb crud operation failed";
+const CF_ERROR: &str = "critical: rocksdb column family operation failed";
+
+/// Ledger sub entry enum
+pub enum LedgerSubEntry {
+    /// Balance
+    Balance,
+    /// Bytecode
+    Bytecode,
+    /// Datastore entry
+    Datastore(Hash),
+}
+
+pub(crate) struct LedgerDB(pub(crate) DB);
+
+/// Destroy the disk ledger db and free the lock
+pub fn destroy_ledger_db() {
+    DB::destroy(&Options::default(), DB_PATH).expect(OPEN_ERROR);
+}
+
+macro_rules! balance_key {
+    ($addr:ident) => {
+        [&[0u8], &$addr.to_bytes()[..]].concat()
```

Comment:
as a good practice let's avoid hardcoded "0u8", "1u8" etc... and use constants declared at the top of the file :)

Reply:
Done.